Gemfile

```diff
@@ -13,7 +13,11 @@ group :test do
   gem 'httpclient', '>= 2.2'
   gem 'mime-types', '~> 1.25', :platforms => [:jruby, :ruby_18]
   gem 'minitest', '>= 5.0.5'
-  gem 'net-http-persistent', '~> 2.9.4'
+  if RUBY_VERSION >= '2.1'
```

Comment:
@mislav This looks like the only working solution and is what it was done already in the past for other gems at the bottom of the Gemfile. Given the Gemfile is only used for tests, I think that is acceptable.